src/registrar/templatetags/custom_filters.py

```diff
@@ -169,3 +169,8 @@ def has_contact_info(user):
         return False
     else:
         return bool(user.title or user.email or user.phone)
+
+
+@register.filter
+def model_name(instance):
+    return instance.__class__.__name__.lower()
```

Comment:
its a possible knowledge gap on my part, but what is a `model_name` in reference to?

Reply:
No fuss, it's totally a new change so dw -- it's used in the `src/registrar/templates/django/admin/includes/detail_table_fieldset.html` file! I'm using it so that when we are clicking into the Portfolio model that the CISA Region won't display, but will still display for the other models that use the file :)